src/LoadingSkeleton/LoadingSkeleton.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Skeleton, { SkeletonTheme } from 'react-loading-skeleton';
+import 'react-loading-skeleton/dist/skeleton.css';
+
+import colors from '../Styles/colors/palette';
+
+const LoadingSkeleton = ({
+ borderRadius,
+ circle,
+ className,
+ containerClassName,
+ containerTestId,
+ count,
+ height,
+ inline,
+ width,
+ ...props
+}) => (
+  <SkeletonTheme baseColor={colors.UX_GRAY_300}>
+    <Skeleton
+      borderRadius={borderRadius}
+      circle={circle}
+      className={classNames('LoadingSkeleton', className)}
+      containerClassName={containerClassName}
+      containerTestId={containerTestId}
+      count={count}
+      height={height}
+      inline={inline}
+      width={width}
+      {...props}
+    />
+  </SkeletonTheme>
+);
+
+LoadingSkeleton.propTypes = {
+  /**
+  The border radius of the skeleton.
+  */
+  borderRadius: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  /**
+  Makes the skeleton circular by setting border-radius to 50%.
+  */
+  circle: PropTypes.bool,
+  className: PropTypes.string,
+  /**
+  A custom class name for the <span> that wraps the individual skeleton elements.
```

Comment:
💅 I'm a little unsure how these comments ultimately get used by Chromatic, but it looks like the `<span>` here is getting interpreted as real html:

<img width="698" alt="Screen Shot 2023-01-04 at 4 10 43 PM" src="https://user-images.githubusercontent.com/3161151/210650613-541244a1-5aaf-448f-9646-c5a2231f6b09.png">



```suggestion
  A custom class name for the `<span>` that wraps the individual skeleton elements.
```

Reply:
Unfortunately I don't think my suggestion did the trick 😢

Reply:
Sorry, I had to build another [Chromatic](https://62d040e741710e4f085e0647-aautmrrylg.chromatic.com/?path=/docs/components-loadingskeleton--default#props) so that was it!